.pre-commit-config.yaml

```diff
@@ -0,0 +1,65 @@
+# Define hooks for code formations
+# Will be applied on any updated commit files if a user has installed and linked commit hook
+
+default_language_version:
+  python: python3.8
+
+# Define bot property if installed via https://github.com/marketplace/pre-commit-ci
+ci:
+  autofix_prs: true
+  autoupdate_commit_msg: '[pre-commit.ci] pre-commit suggestions'
+  autoupdate_schedule: quarterly
+  # submodules: true
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+      - id: check-case-conflict
+      - id: check-yaml
+      - id: check-json
+      - id: check-docstring-first
+
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v2.23.1
+    hooks:
+      - id: pyupgrade
+        args: [--py36-plus]
+        name: Upgrade code
+
+  - repo: https://github.com/PyCQA/isort
```

Comment:
Could replace this with:

```
  - repo: https://github.com/omnilib/ufmt
    rev: v1.3.0
    hooks:
      - id: ufmt
        additional_dependencies:
          - black == 21.9b0
          - usort == 0.6.4
```
which does both the sorting and formatting with black (which is what we've been using so far)

Reply:
yes, sure if you want to go with Back :+1: